src/handles/handleDiagnostic.ts

```diff
@@ -13,6 +13,7 @@ import { ILinterConfig, SecurityKey } from '../common/types';
 import { executeFile, findWorkDirectory, findCommand, checkAnyFileExists } from '../common/util';
 import HunkStream from '../common/hunkStream';
 import logger from '../common/logger';
+import lodashGet from 'lodash/get';
```

Comment:
I usually stay away from lodash, but it's `get` function is great for what it's used for here.